src/algorithms/calorimetry/CalorimeterHitDigi.cc

```diff
@@ -136,7 +137,7 @@ void CalorimeterHitDigi::AlgorithmProcess()  {
 void CalorimeterHitDigi::single_hits_digi(){
 
      // Create output collections
-    for ( auto ahit : simhits ) {
+    for ( const auto *ahit : simhits ) {
```

Comment:
I expect a conflict with #741. Collections return objects (with pointers inside), not references or pointers.